src/services/utils.js

```diff
@@ -5,139 +5,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 import { store } from '../redux/store';
-
-export const FetchStatus = {
-    SUCCEED: 'SUCCEED',
-    FAILED: 'FAILED',
-    IDLE: 'IDLE',
-    RUNNING: 'RUNNING',
-};
-
-export const getWsBase = () =>
-    document.baseURI
-        .replace(/^http:\/\//, 'ws://')
-        .replace(/^https:\/\//, 'wss://');
-
-export const getRequestParamFromList = (params, paramName) => {
-    return new URLSearchParams(
-        params?.length ? params.map((param) => [paramName, param]) : []
-    );
-};
-
-const parseError = (text) => {
-    try {
-        return JSON.parse(text);
-    } catch (err) {
-        return null;
-    }
-};
-
-const handleError = (response) => {
-    return response.text().then((text) => {
-        const errorName = 'HttpResponseError : ';
-        let error;
-        const errorJson = parseError(text);
-        if (
-            errorJson &&
-            errorJson.status &&
-            errorJson.error &&
-            errorJson.message
-        ) {
-            error = new Error(
-                errorName +
-                    errorJson.status +
-                    ' ' +
-                    errorJson.error +
-                    ', message : ' +
-                    errorJson.message
-            );
-            error.status = errorJson.status;
-        } else {
-            error = new Error(
-                errorName +
-                    response.status +
-                    ' ' +
-                    response.statusText +
-                    ', message : ' +
-                    text
-            );
-            error.status = response.status;
-        }
-        throw error;
-    });
-};
+import { fetchAppsAndUrls, fetchEnv } from '@gridsuite/commons-ui';
 
 export const getToken = () => {
```

Comment:
It's still needed ?

Reply:
Well it's the point of the ticket, mutualize what we can, it won't be done after if we don't do it now, i'm pretty sure, so well...
You can check with a TL but for me, it need to be done here alos.

Reply:
I've just talked about it with @TheMaskedTurtle, we ought to prioritize fixing commons-ui to release gridstudy deadlock.
I can create a following ticket to clean up  the rest of utils functions getVersion, auth function, etc...

Reply:
This said I can clean up `getToken` since it's already in commons ui, didn't noticed earlier

Reply:
Yes I figured this too later, fine by me I'll do a second PR later then